DESCRIPTION

```diff
@@ -13,8 +13,18 @@ BugReports: https://github.com/Bioconductor/GenomicAlignments/issues
 Version: 1.33.1
 License: Artistic-2.0
 Encoding: UTF-8
-Author: Hervé Pagès, Valerie Obenchain, Martin Morgan
-Maintainer: Bioconductor Package Maintainer <maintainer@bioconductor.org>
+Authors@R: c(
+    person("Hervé", "Pages", role = "aut"),
+    person("Valerie", "Obenchain", role="aut"),
+    person("Martin", "Morgan", role="aut"),
+    person("Halimat C.", "Atanda", role = "ctb",
+        comment = "Translated 'WorkingWithAlignedNucleotides' vignette from Sweave to RMarkdown / HTML."
+    ),
+    person(
+        "Bioconductor Package Maintainer",
+        role = "cre"
+        email = "maintainer@bioconductor.org"
+    ))
```

Comment:
The `DESCRIPTION` should follow what's currently in the master branch:

```
Authors@R: c(
	person("Hervé", "Pagès", role=c("aut", "cre"),
		email="hpages.on.github@gmail.com"),
	person("Valerie", "Obenchain", role="aut"),
	person("Martin", "Morgan", role="aut"))
```
Note: it should also use the same indent and spacing around `=`. Herve is the maintainer denoted by `cre` in `role`. You should add your person entry to this following the same format. This will also resolve the merge conflict.

Reply:
Hi @jwokaty Should I remove the entry of Bioconductor package maintainer?

Reply:
Yes, since Herve is the maintainer.

Reply:
All done